src/components/AddItemsForm.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import { items } from '../lib/firebase';
+import '../css/components/AddItemsForm.css';
+
+const AddItemsForm = () => {
+  let [inputValue, setInputValue] = useState('');
+
+  const addToDatabase = (e) => {
+    e.preventDefault();
+    items()
```

Comment:
Here, I think you could wrap the code from lines 10-18 in a 'try' block before the 'catch' block on line 19

Reply:
Hi @yahayakenny thanks for your review ✨
and It's a promise, not a try/catch block

Reply:
Hi Sarah, yeah, I just went ahead and read a bit about it and also discovered that. You guys have some really great work here. Everything works perfectly. Good job  👍